examples/annotation_import/tiled.ipynb

```diff
@@ -20,12 +20,12 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    import cv2
</pre><p> </p><p>These are duplicates. I would just add all imports at the top.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/817/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Done</p>